index.bs

```diff
@@ -4902,6 +5009,56 @@ as candidates to be employed in a [=registration ceremony=].
 : Authenticator extension output
 :: None.
 
+
+## Credential Properties Extension (credProps) ## {#sctn-authenticator-credential-properties-extension}
+
+This extension allows [=[WRPS]=] to query additional properties of the [=public key credential source=] created during a 
+[=registration ceremony=] or used during an [=authentication ceremony=].
+
+: Extension identifier
+:: `credProps`
+
+: Operation applicability
+:: [=registration extension|Registration=] and [=authentication extension|Authentication=]
+
+: Client extension input
+:: The Boolean value [TRUE] to indicate that this extension is requested by the [=[RP]=].
+    <xmp class="idl">
+    partial dictionary AuthenticationExtensionsClientInputs {
+        boolean credProps;
+    };
+    </xmp>
+
+: Client extension processing
+:: None, except creating the authenticator extension input from the client extension input.
+
+: Client extension output
+:: Returns the credential properties as discovered by the client, if any, such as when a platform authenticator is used. 
+    The client MAY omit any properties at its discretion, and SHOULD omit any properties that would be duplicated in the
+    authenticator extension output. This dictionary is initially defined with one credential property 
+    {{CredentialPropertiesOutput/rk}} to indicate if the authenticator has created or used a resident key 
+    credential. Other credential properties MAY be returned in this extension.
```

Comment:
&ldquo;Other credential properties MAY be returned in this extension&rdquo;, but they aren't defined. This appears to be giving implementations permission to invent new properties at will. I don't necessarily disagree with doing that but, if that's the intention, I believe it should be called out explicitly rather than hinted at.

Reply:
Modified wording to indicate that it's at the discretion of the client if they wish to add additional properties.